app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -259,6 +259,12 @@ export class ParsedCommand {
                     this.i++;
                     break;
                 }
+                case '—': {
```

Comment:
wrt to "introduced when two '-' are set in iOS" ? - is it IOS only? (can they generate the em dash on other platforms as well?)

Reply:
The em dash can be created in any platform (worst case scenario, copy/paste). The problem with iOS is that by default, when you type to dashes, it automatically convert that to an em dash. Therefore why we have to deal with this case.

Reply:
But the code here is platform-agnostic, right?- meaning that if someone would ever type in the emdash directly we would address it as below (I understand that on iOS we get the emdash when the user types the "--" which is the more likely scenario to get the emdash).

Reply:
Yes. The code is platform agnostic, and for simplicity, will be the same for webapp, so any version of the apps (Android, iOS, Webapp, Desktop...) will use an em dash as a double dash.